lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -286,6 +287,27 @@ class ReportUIFeatures {
     this._dom.resetTemplates();
   }
 
+  /**
+   * Jumps to respective audit on page load if hash is provided in URL
+   * Collapsed content is opened automatically
+   * Element is highlighted via CSS selector
+   */
+  _jumpToAudit() {
+    const hash = window.location.hash;
+    if (hash) {
+      const audit = document.querySelector(hash);
+      if (audit) {
+        if (audit.firstElementChild) {
+          audit.firstElementChild.setAttribute('open', 'true');
+        }
+        if (audit.parentElement) {
+          audit.parentElement.setAttribute('open', 'true');
+          scroll({top: audit.parentElement.offsetTop, behavior: 'smooth'});
```

Comment:
can you call it as `window.scroll(` to be clear its the global method. I'm not used to seeing it so nude. :)